src/shared/credentials/defaultCredentialSelectionDataProvider.ts

```diff
@@ -40,15 +40,15 @@ export class DefaultCredentialSelectionDataProvider implements CredentialSelecti
         this._credentialsMru = new CredentialsProfileMru(context)
     }
 
-    public async pickCredentialProfile(
+    public async pickCredentialSource(
         input: MultiStepInputFlowController,
         state: Partial<CredentialSelectionState>
     ): Promise<QuickPickItem> {
         return await input.showQuickPick({
-            title: localize('AWS.title.selectCredentialProfile', 'Select an {0} credential profile', getIdeProperties().company),
+            title: localize('AWS.title.selectCredentialSource', 'Select a {0} credential source', getIdeProperties().company),
             step: 1,
             totalSteps: 1,
-            placeholder: localize('AWS.placeHolder.selectProfile', 'Select a credential profile'),
+            placeholder: localize('AWS.placeHolder.selectSource', 'Select a credential source'),
```

Comment:
Is "profile" more intuitive for the common case? User is choosing a source:profile pair.